carsus/io/output/ionization_energies.py

```diff
@@ -0,0 +1,38 @@
+import copy
+
+class IonizationEnergiesPreparer:
+    def __init__(self, cmfgen_reader, ionization_energies):
+        self.ionization_energies = ionization_energies
+
+        if (cmfgen_reader is not None) and hasattr(
+            cmfgen_reader, "ionization_energies"
+        ):
+            combined_ionization_energies = copy.deepcopy(ionization_energies)
+            combined_ionization_energies.base = (
+                cmfgen_reader.ionization_energies.combine_first(
+                    ionization_energies.base
+                )
+            )
+            self.ionization_energies = combined_ionization_energies
```

Comment:
Cross comparing the original code 
https://github.com/tardis-sn/carsus/blob/bf6636a3b20a226817f3abb6d4f43902ced8c108/carsus/io/output/base.py#L75-L76
Is the else condition being removed on purpose?

Reply:
Yes- it doesn't do anything because `self.ionization_energies = ionization_energies` is defined at the top of the function. The cmfgen clause just overwrites it.

Reply:
ohhhhh good catch! My bad didn't see that ><